### DISCUSSION TITLE
loading color Replace white to black

### DISCUSSION THREAD

**Discussion by tkdwns1733** [google/model-viewer#2160] [Q&A]
### Description

First of all, main background color is black.

During the 3D loading delay time, a white background comes out behind.

I'd like to deal with this issue naturally.

Transparent, gray comes out.

Is there a way to make white areas appear black?

![스크린샷 2021-03-30 00 57 09](https://user-images.githubusercontent.com/68719235/112864608-ed245a00-90f2-11eb-819c-b610708a0ec2.png)

That color comes out during loading time.

I need black or transparent..

### Version

- model-viewer: vX.X.X

### Browser Affected

- [o] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [o] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [ ] QuickLook[

](url)

**Comment by elalish** [comment]
I don't quite understand what you mean, but have you looked at setting poster-color to transparent? https://modelviewer.dev/docs/#entrydocs-loading-css-posterColor

**Comment by tkdwns1733** [comment]
sorry, my english not good.
 
Attach the video. 
 
During loading tile, that's the background of the model-viewer.
 
i want without distinction be black
 
-----Original Message-----
